addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -15,6 +15,9 @@ const {
     dom: { triggerEvent },
 } = require('web.test_utils');
 
+const Bus = require('web.Bus');
+const testUtils = require('web.test_utils');
```

Comment:
`test_utils` from web is already imported just above, please add keys you need to it, and also **b**us comes alphabetically before **t**est_utils